linux_os/guide/services/sssd/sssd_certificate_verification/tests/wrong_value_not_default.fail.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+# packages = sssd-common
+# variables = var_sssd_certificate_verification_digest_function=sha512
+
+mkdir -p /etc/sssd/conf.d
+touch /etc/sssd/sssd.conf
+echo -e "[sssd]\ncertificate_verification = ocsp_dgst=sha256" >> /etc/sssd/sssd.conf
```

Comment:
Should this be a variable as well?

Reply:
You mean if `ocsp_dgst` should be set using the variable? No

The test's goal is to setup a system with a wrong value and have an expected XCCDF value that is not the default one.